pentaho-karaf-assembly/src/main/filtered-resources/etc/custom.properties

```diff
@@ -109,8 +109,8 @@ org.apache.commons.logging.*; version\="1.1.3", \
  org.pentaho.xul.swt.tab, \
  org.pentaho.platform.*, \
  org.slf4j.*; version\="1.7.7", \
- org.springframework.dao; version\="4.3.2.RELEASE", \
- org.springframework.security.*; version\="4.1.5.RELEASE", \
+ org.springframework.dao; version\="4.3.19.RELEASE", \
+ org.springframework.security.*; version\="4.2.9.RELEASE", \
```

Comment:
Please use maven filtering to fill in the version value.

Reply:
Agree, working on it